frontend/src/components/HeaderUser.tsx

```diff
@@ -0,0 +1,58 @@
+import "../index.css";
+import React, { useState } from "react";
+import { SNOWPACK_PUBLIC_API_URL } from "../model";
+
+const logout = async () => {
+  await fetch(`${SNOWPACK_PUBLIC_API_URL}/api/logout`, {
+    method: "POST",
+  })
+    .then((res) => {
+      if (res.ok) {
+        return true;
+      } else {
+        throw new Error("Could not log out.");
+      }
+    })
+    .catch((error) => {
+      console.log(error);
+      return false;
+    });
+  // Redirect to login page
+  window.location.href = "/";
+};
+
+const showSettings = () => {
+  return (
+    <div class="settings-list" aria-labelledby="dropdownMenuLink">
+      <a class="dropdown-item" onClick={logout}>
```

Comment:
It should be `className` instead of `class` in both cases. (That's how the property is called in JSX).